docs/source/executing-operations/demand-control.mdx

```diff
@@ -0,0 +1,577 @@
+---
+title: Demand Control
+subtitle: Protect your graph from high-cost GraphQL operations
+description: Protect your graph from malicious or demanding clients with Apollo GraphOS Router's demand control features. Estimate, calculate, observe, and reject high-cost GraphQL operations.
+minVersion: 1.48.0
+---
+
+<EnterpriseFeature />
+
+<PreviewFeature>
+
+The Demand Control feature is in [preview](/resources/product-launch-stages/#product-launch-stages) for organizations with an Enterprise plan. Get in touch with your Apollo contact to request access.
```

Comment:
This is still inaccurate. They don't need to get in touch with their Apollo contact.

Reply:
```suggestion
The Demand Control feature is in [preview](/resources/product-launch-stages/#product-launch-stages) for organizations with an Enterprise plan.
```